lib/wallet/http.js

```diff
@@ -1035,6 +1035,41 @@ class HTTP extends Server {
       return res.json(200, mtx.getJSON(this.network));
     });
 
+    // Create Batch Open
+    this.post('/wallet/:id/batch/open', async (req, res) => {
+      const valid = Validator.fromRequest(req);
+      const names = valid.array('names');
+      const force = valid.bool('force', false);
+      const passphrase = valid.str('passphrase');
+      const broadcast = valid.bool('broadcast', true);
+      const sign = valid.bool('sign', true);
+      const MAX_NAME_ARRAY_LENGTH = 200;
+
+      assert(names && names.length > 0, 'Names are required.');
+      assert(names.length <= MAX_NAME_ARRAY_LENGTH,
+        `Names array shoud not exceed ${MAX_NAME_ARRAY_LENGTH}`);
+      assert(broadcast ? sign : true, 'Must sign when broadcasting.');
+
+      const options = TransactionOptions.fromValidator(valid);
+      const {mtx, errors, isAllError} = await req.wallet
+      .createBatchOpen(names, force, options);
+
+      if (isAllError) { // no valid output in mtx
```

Comment:
Personally I don't like how `hsd` eslint rules want:
```javascript
if (isAllError)
  return res.json(500, { errors: errors });
```

I think this --^ is very prone to bugs and don't like it, I prefer what you wrote, but, it's the convention in this repo, so could you change this if statement to that format? This should happen automatically with the rest of the eslint fixes in this PR